ios/RNTrackPlayer/Vendor/SwiftAudio/Classes/RemoteCommandController/RemoteCommandController.swift

```diff
@@ -30,9 +30,11 @@ public class RemoteCommandController {
     }
     
     internal func enable(commands: [RemoteCommand]) {
-        self.disable(commands: RemoteCommand.all())
-        commands.forEach { (command) in
-            self.enable(command: command)
+        RemoteCommand.all().forEach { (command) in
+            self.disable(command: command)
```

Comment:
Can this line  be placed in the else branch, otherwise, the control buttons will still flicker